src/registrar/templates/application_other_contacts.html

```diff
@@ -13,38 +13,69 @@
       <legend>
-        <h2>Organization contact {{ forloop.counter }} (optional)</h2>
+        <h2>Are there other employees who can help verify your request?</h2>
       </legend>
 
-      {% input_with_errors form.first_name %}
+      {% with add_class="usa-radio__input--tile" add_legend_class="usa-sr-only" %}
+        {% input_with_errors forms.0.has_other_contacts %}
+      {% endwith %}
+      {# forms.0 is a small yes/no form that toggles the visibility of "other contact" formset #}
 
-      {% input_with_errors form.middle_name %}
+    </fieldset>
 
-      {% input_with_errors form.last_name %}
+    <div id="other-employees">
+        {% include "includes/required_fields.html" %}
+        {{ forms.1.management_form }}
+        {# forms.1 is a formset and this iterates over its forms #}
+        {% for form in forms.1.forms %}
+            <fieldset class="usa-fieldset">
+            <legend>
+                <h2>Organization contact {{ forloop.counter }} (optional)</h2>
+            </legend>
 
-      {% input_with_errors form.title %}
+            {% input_with_errors form.first_name %}
 
-      {% input_with_errors form.email %}
+            {% input_with_errors form.middle_name %}
 
-      {% with add_class="usa-input--medium" %}
-        {% input_with_errors form.phone %}
-      {% endwith %}
+            {% input_with_errors form.last_name %}
 
-    </fieldset>
-  {% endfor %}
+            {% input_with_errors form.title %}
+
+            {% comment %} There seems to be an issue with the character counter on emails.
+            It's not counting anywhere, and in this particular instance it's
+            affecting the margin of this block. The wrapper div is a 
+            temporary workaround. {% endcomment %}
+            <div class="margin-top-3">
+                {% input_with_errors form.email %}
+            </div>
+
+            {% with add_class="usa-input--medium" %}
+                {% input_with_errors form.phone %}
+            {% endwith %}
+
+            </fieldset>
+        {% endfor %}
+
+        <button type="submit" name="submit_button" value="save" class="usa-button usa-button--unstyled">
+            <svg class="usa-icon" aria-hidden="true" focusable="false" role="img" width="24" height="24">
+            <use xlink:href="{%static 'img/sprite.svg'%}#add_circle"></use>
+            </svg><span class="margin-left-05">Add another contact</span>
+        </button>
+    </div>
 
-  <button type="submit" name="submit_button" value="save" class="usa-button usa-button--unstyled">
-    <svg class="usa-icon" aria-hidden="true" focusable="false" role="img" width="24" height="24">
-      <use xlink:href="{%static 'img/sprite.svg'%}#add_circle"></use>
-    </svg><span class="margin-left-05">Add another contact</span>
-  </button>
+    <div id="no-other-employees">
+        <fieldset class="usa-fieldset margin-top-2">
+          <legend>
+            <h2>No other employees from your organization?</h2>
```

Comment:
**Question:** I think you might be missing text here per the mock up? 

`You don't need to provide names of other employees now, but it may slow down our assessment of your eligibility. Describe why there are no other employees who can help verify your request`

<img width="566" alt="Screenshot 2024-01-10 at 3 59 19 PM" src="https://github.com/cisagov/manage.get.gov/assets/13954664/f505fc24-e074-432f-aa7d-c7f7c25424c4">

Reply:
Maybe I'm nitpicking, but it seems odd to me in the radio button we ask them to explain why, but we don't require them.... should we be requiring them to explain why?